pkg/apis/networking/types.go

```diff
@@ -138,13 +138,17 @@ type NetworkPolicyPort struct {
 	// +optional
 	Protocol *api.Protocol
 
-	// The port on the given protocol. This can either be a numerical or named port on
-	// a pod. If this field is not provided, this matches all port names and numbers.
+	// The port on the given protocol. This can either be a numerical or named
+	// port on a pod. If this field is not provided, this matches all port names and
+	// numbers, whether an endPort is defined or not.
```

Comment:
no, specifying `endPort` without `port` should fail validation

Reply:
Right! Removed the end of the comment, and commented in endPort as:

// EndPort defines the end of the port range, being the end included within the
	// range. This field cannot be defined if the port field is not defined or if the
	// port field is defined as a named port.